### ISSUE TITLE
no domain in Nansat.reproject

### ISSUE THREAD

**Issue by aleksandervines** [nansencenter/nansat#158]
```
def reproject(self, dstDomain=None,...):
        if dstDomain is None:
            return
```

This seems illogical. Why do we provide dstDomain with a defaul value of None if we only return on it. Why not let Python send the usual TypeError if the argument is missing? (of course, we still have to handle if the user actually sends in None as dstDomain)

**Comment by akorosov** [comment]
He-he. That's a legacy of old version without undo. Should be optimized, of course.

**Comment by akorosov** [comment]
Closed in 127a84fd34674f5f7501ec86876f75068cec7fe7
